### DISCUSSION TITLE
Add context to the"Reset" option buttons

### DISCUSSION THREAD

**Discussion by ichnilatis-gr** [koreader/koreader#10479] [Q&A] [answered]
Dear @Frenzie I need your help.
There are times when the word "reset" in a question needs to be translated with a different word in Greek each time, so the text of the button should be translated accordingly.
Could you, please, instruct me how to add a different translation for the ["reset" button](https://hosted.weblate.org/translate/koreader/koreader/en/?checksum=acaa632ead88db3f) in these strings? 

- ["Do you want to reset statistics for day %1 for this book?"](https://hosted.weblate.org/translate/koreader/koreader/en/?checksum=6dc1c80159ff9d38)
- ["Do you want to reset statistics for this period for book: %1"](https://hosted.weblate.org/translate/koreader/koreader/en/?&offset=3181)

You [explained it to me once with pictures](https://github.com/koreader/koreader/pull/8489#issuecomment-986029073), but I can't understand how I can do a PR in this case.

Thank you!

**Comment by Frenzie** [comment]
Ping me next week if I forget please.

**Reply by Frenzie** [reply]
Taking a bit more time to read your message, you have to click "commit" in the top right after making your edits. The rest is hopefully fairly self-explanatory.

<img src=https://github.com/koreader/koreader/assets/202757/d2af8918-448c-424f-8d3e-ec927e3eab7b width=60%>

Then you can also add a bit more explanation there, but creating a PR should be more or less automatic.
<img src=https://github.com/koreader/koreader/assets/202757/84194655-5d1f-4e58-8e01-9cd8c135d1ef width=60%>

**Reply by ichnilatis-gr** [reply]
(After a long time...) Dear @Frenzie , my problem is that I don't know exactly what to write. For example, you had made this change: 

![εικόνα](https://github.com/koreader/koreader/assets/76203123/982d8fd4-1bab-41a1-b653-46326fbb83c9)

Now, how can I adapt this coding in this case?

**Reply by Frenzie** [reply, marked as answer]
Unfortunately the answer is with some more difficulty than I anticipated because `ReaderStatistics:resetStatsForBookForPeriod()` is being clever.

I think this should do the trick? https://github.com/koreader/koreader/pull/10621

**Reply by ichnilatis-gr** [reply]
Thank you very much!
